cves/cve_2017_5754.py

```diff
@@ -79,14 +79,14 @@ def validation_flow_chart():
     """This function creates a graph that shows the vulnerability validation process of Meltdown."""
     meltdown_path = '/sys/devices/system/cpu/vulnerabilities/meltdown'
     vol_graph = graphviz.Digraph('G', filename=VULNERABILITY, format='png')
```

Comment:
Why isn't: graphviz.Digraph() also in the `graph_functions.py` file?
(This is true for all vulnerabilities)